public/app/plugins/datasource/loki/modifyQuery.ts

```diff
@@ -177,9 +177,9 @@ function getLineFiltersPositions(query: string): Position[] {
   const tree = parser.parse(query);
   const positions: Position[] = [];
   tree.iterate({
-    enter: (type, from, to, get): false | void => {
-      if (type.name === 'LineFilters') {
-        positions.push({ from, to });
+    enter: ({ node }): false | void => {
+      if (node.name === 'LineFilters') {
+        positions.push({ from: node.from, to: node.to });
```

Comment:
I am a bit unsure of the parameters we use in the `enter`-method now.
With those changes we will have different approaches:
1. `enter: ({ type, from, to }): false | void`
2. `enter: ({ name, from, to }): false | void`
3. `enter: ({ node }): false | void`
4. `enter: ({ name, from, to, node }): false | void`
5. `enter: (nodeRef): false | void`
6. `enter: (node): false | void `

Even though each of them might be leading to the correct outcome, I'd prefer to have the same signature (where the logic is the same) on each function. What do you think?

Reply:
I've cleaned up the last few calls in this file that weren't destructuring the `SyntaxNodeRef`, so now all parameters from the enter method should all have a consistent format, let me know if this is what you were looking for or not!